verifact/forum/schema.py

```diff
@@ -39,3 +40,87 @@ def resolve_questions(root, info):
 
     def resolve_answers(root, info):
         return Answer.objects.all()
+
+
+# MUTATIONS
+
+
+class CreateQuestion(Mutation):
+    class Arguments:
+        created_at = DateTime()
```

Comment:
Should `created_at` be configurable? Or automatically set by the backend?

Reply:
this (and all the "why is this an argument"-related comments below) expose that this is my first time with Django - oops! I think when I first wrote this I thought the arguments had to be the same as the model's attributes... not sure why that didn't click when I added a question pk vs a question though.. anyways dealing with all of these, thanks!

Reply:
No worries! I can see how that could be confusing this being your first time working with Django and presumably also with GraphQL.